plugin/src/main/java/com/denizenscript/denizen/events/block/MoistureChangeScriptEvent.java

```diff
@@ -0,0 +1,91 @@
+package com.denizenscript.denizen.events.block;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.block.data.type.Farmland;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.MoistureChangeEvent;
+
+public class MoistureChangeScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // farmland moisture level changes
+    //
+    // @Group Block
+    //
+    // @Location true
+    //
+    // @Triggers when a farmland moisture level changes
```

Comment:
Maybe `farmland's`? or `farmland block's`? also most events seem to have a `.` at the end